server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -422,6 +423,12 @@ public class ShuffleServerConf extends RssBaseConf {
               "The interval of trigger shuffle buffer manager to flush data to persistent storage. If <= 0"
                   + ", then this flush check would be disabled.");
 
+  public static final ConfigOption<ShuffleBufferType> SERVER_SHUFFLE_BUFFER_TYPE =
```

Comment:
If we can always achieve good results, why do we still need this switch?

If we can not always achieve good results, please elaborate more in the description. Let users know when it is suitable to set the type to SKIP_LIST.

Reply:
> If we can always achieve good results, why do we still need this switch?

The `slow start` feature of MR requires that the order of blocks can't be changed. https://github.com/apache/incubator-uniffle/issues/1708#issuecomment-2117233393

Reply:
Then, you can add this into the description, maybe also with some detailed explanations & suggestions about this config.

Reply:
Done